sphinx/source/conf.py

```diff
@@ -86,6 +86,10 @@
 
 autodoc_member_order = 'groupwise'
 
+autodoc_type_aliases = {
+    'ArrayLike': 'ArrayLike',  # This avoids complicated Unions in generated docs
```

Comment:
This is needed for the `contour` docstring otherwise `ArrayType` is expanded out to unpleasantly long `Union` of the possible types that are convertable to a numpy array.

Reply:
This makes sense.